packages/core-database-postgres/lib/connection.js

```diff
@@ -351,7 +351,7 @@ module.exports = class PostgresConnection extends ConnectionInterface {
    * @param  {Block} block
    * @return {void}
    */
-  async saveBlockAsync (block) {
+  saveBlockAsync (block) {
```

Comment:
Should be renamed and the doc updated, it doesn't save anything. Maybe something like `queueSaveBlockAsync`?

Reply:
even better